cypress/e2e/spec.cy.ts

```diff
@@ -0,0 +1,63 @@
+describe('Book ticket', () => {
+	it('visit site and book a ticket as a guest', () => {
+		// visit the site and click on "filmer" (movies)
+		cy.visit('/');
+		cy.get('nav li:first').click();
+
+		// click the first movie
+		cy.get('ul li:first').click();
+
+		// click the first screening
+		cy.get('[data-testid="book-screening-btn"]:first').click();
+
+		// choose number of tickets, seat and click guest button
+		cy.wait(2000);
```

Comment:
This should generally not be necessary. The getter functions should wait implicitly (until a timeout is surpassed).